src/frontend/screens/PrivacyPolicy/index.tsx

```diff
@@ -0,0 +1,307 @@
+    defaultMessage:
+      'Such as model and manufacturer of your device; device operating system; screen size; device locale (language); device memory.',
+  },
+  appInfo: {
+    id: 'screens.PrivacyPolicy.appInfo',
+    defaultMessage: 'App Info',
+  },
+  appInfoDescription: {
+    id: 'screens.PrivacyPolicy.appInfoDescription',
+    defaultMessage: 'The version and locale (language) of CoMapeo',
+  },
+  appUsageTitle: {
+    id: 'screens.PrivacyPolicy.appUsageTitle',
+    defaultMessage: 'App Usage',
+  },
+  userCount: {
+    id: 'screens.PrivacyPolicy.userCount',
+    defaultMessage: 'User Counts',
+  },
+  userCountDescription: {
+    id: 'screens.PrivacyPolicy.userCountDescription',
+    defaultMessage:
+      'The number of users per country and per project. Aggregated and anonymized',
+  },
+  whyCollected: {
+    id: 'screens.PrivacyPolicy.whyCollected',
+    defaultMessage: 'Why is this data collected?',
+  },
+  whyCollectedDescription: {
+    id: 'screens.PrivacyPolicy.whyCollectedDescription',
+    defaultMessage:
+      'Crash data and app errors together with the device and app info provide Awana Digital with the information we need to fix errors and bugs in the app. The performance data helps us improve the responsiveness of the app and identify errors. User counts, including total users, users per country, and users per project, help justify ongoing investment in the development of CoMapeo.',
+  },
+  notCollected: {
+    id: 'screens.PrivacyPolicy.notCollected',
+    defaultMessage: 'What is not collected?',
+  },
+  notCollectedDescription: {
+    id: 'screens.PrivacyPolicy.notCollectedDescription',
+    defaultMessage:
+      'We do not collect any personal data or anything that can be used to identify or track a user or device. Device identifiers used to aggregate information are random, anonymous, and changed every month.\n\nDiagnostic information does not include data about how you use the app, and it never includes any of the data you have collected with the app. We do not collect your precise or coarse location, only the country where you are using CoMapeo.',
+  },
+  thirdParty: {
+    id: 'screens.PrivacyPolicy.thirdParty',
+    defaultMessage: 'Third-party access to data',
+  },
+  thirdPartyDescription: {
+    id: 'screens.PrivacyPolicy.thirdPartyDescription',
+    defaultMessage:
+      'A "third-party" is an organization other than Awana Digital.',
+  },
+});
+
+type PrivacyPolicyProps = Partial<
+  NativeStackScreenProps<DeviceNamingParamsList, 'OnboardingPrivacyPolicy'>
+>;
+
+export const PrivacyPolicy: React.FC<PrivacyPolicyProps> = () => {
+  const {formatMessage} = useIntl();
+  const [awanaExpanded, setAwanaExpanded] = React.useState(false);
+  const [openSourceExpanded, setOpenSourceExpanded] = React.useState(false);
+
+  const toggleAboutAwana = () => setAwanaExpanded(!awanaExpanded);
+  const toggleOpenSource = () => setOpenSourceExpanded(!openSourceExpanded);
```

Comment:
use the functional update syntax

```suggestion
  const toggleAboutAwana = () => setAwanaExpanded(prev => !prev);
  const toggleOpenSource = () => setOpenSourceExpanded(prev => !prev);
```

Reply:
nit: not really necessary to define named functions for this as opposed to inlining in the relevant press handlers (since they're each only used once)